.github/workflows/run-enterprise-action.yml

```diff
@@ -28,8 +32,11 @@ jobs:
         id: enterprise-action
         uses: ./
         with:
-          milliseconds: ${{ inputs.milliseconds }}
-          fail: ${{ inputs.fail }}
+          api_token: ${{ secrets.TEST_CLOUD_API_TOKEN }}
```

Comment:
We can configure a token in this repo's secrets to allow manual testing of the action.